tests/lib/rules/no-func-assign.js

```diff
@@ -88,6 +88,15 @@ ruleTester.run("no-func-assign", rule, {
                 data: { name: "foo" },
                 type: "Identifier"
             }]
+        },
+        {
+            code: "var a = function foo() { foo = 123; };",
+            parserOptions: { ecmaVersion: 6 },
```

Comment:
`parserOptions` seems to be unnecessary in this test case. It might be better to remove it, to avoid confusion.

Reply:
```suggestion
```